src/Mapbender/CoreBundle/Resources/public/mapbender.model.ol4.js

```diff
@@ -434,7 +434,9 @@ Mapbender.Model.prototype.createVectorLayer = function(options, owner) {
     var uuid = Mapbender.UUID();
     this.vectorLayer[owner] = this.vectorLayer[owner] || {};
     options.map = this.map;
-    options.style = options.style ? this.createVectorLayerStyle(options.style) : this.createVectorLayerStyle();
+    options.style = !(options.style instanceof ol.style.Style) ? this.createVectorLayerStyle(options.style) :
```

Comment:
Please save logic expression in a separated variable for better readability e.G. var isOl4Styl = options.style instanceof ol.style.Style